packages/bot-web-ui/src/stores/flyout-help-store.js

```diff
@@ -28,7 +28,7 @@ export default class FlyoutHelpStore {
         const block_hw        = Blockly.Block.getDimensions(block_node);
         const block_type      = block_node.getAttribute('type');
         const title           = Blockly.Blocks[block_type].meta().display_name;
-        const help_string_obj = await import(/* webpackChunkName: `[request]` */ `../scratch/help-content/help-strings/${block_type}`);
+        const help_string_obj = await import(/* webpackChunkName: `[request]` */ '@deriv/bot-engine');
```

Comment:
Will this work?

Reply:
@aaron-binary you are right, it wont work, i updated the code